scripts/merge_truth_per_tract.py

```diff
@@ -187,20 +215,34 @@ def main():
     parser.add_argument("--validate", action="store_true")
     parser.add_argument("--silent", action="store_true")
     parser.add_argument("--matching-only", action="store_true")
+    parser.add_argument("--tracts", nargs="+", type=int, help="List of tract numbers to run. Use to format input_dir and object_catalog_path")
+    parser.add_argument("--tract-list", help="File contains a list of tract numbers to run. Use to format input_dir and object_catalog_path")
+    parser.add_argument("--n-cores", type=int, default=1)
 
     args = parser.parse_args()
-
-    if not args.matching_only:
-        df = merge_truth_per_tract(**vars(args))
-
-    if args.object_catalog_path:
-        if args.matching_only:
-            df = args.input_dir
-        df = match_object_with_merged_truth(df, args.object_catalog_path, **vars(args))
-    elif args.matching_only:
-        parser.error("--object-catalog-path must be specified when --matching-only is set.")
-
-    save_df_to_disk(df, **vars(args))
+    kwargs = vars(args)
+
+    if args.tract_list or args.tracts:
```

Comment:
args.tracts is necessarily non empty if nargs="+" in the parser argument definition, no?